recipes/quota_anchor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "quota_anchor" %}
+{% set version = "0.0.1a0" %}
+{% set tag = "v0.0.1a0" %}
+{% set sha256 = "bc0b6fd4410e3c7461686362d64e53186db51cae8888612694d2af1326e977be" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/baoxingsong/quota_Anchor/archive/refs/tags/{{ tag }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - muscle {{ pin_compatible("muscle", max_pin="5") }}
```

Comment:
Please add `pin_subpackage` as described in the docs. Are you sure you need this `pin_compatible`?
https://bioconda.github.io/contributor/linting.html#missing-run-exports
> Importantly note that this shall not be used to pin compatible versions of other recipes in the current recipe. Rather, those other recipes have to get their own run_exports sections. Usually, there is no need to use pin_compatible, just use pin_subpackage as shown above, and fix run_exports in upstream packages as well if needed.